src/service/performance-impl.js

```diff
@@ -230,7 +230,15 @@ export class Performance {
       list.getEntries().forEach(processEntry);
       this.flush();
     });
-    observer.observe({entryTypes: entryTypesToObserve});
+
+    // Wrap observer.observe() in a try statement for testing, because
+    // Webkit throws an error if the entry types to observe are not natively
```

Comment:
so we're OK to not measure safari traffic?

Reply:
We won't be able to read metrics from Performance APIs that Webkit hasn't implemented yet like EventTiming and LayoutJank, but when they do we will start getting numbers.